### PR TITLE
Yandex.Metrica destination documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Documentation for Yandex.Metrica destination

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/yandex-metrica.mdx [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+title: "Yandex.Metrica"
+description: >-
+  Step-by-step guide on sending your event data from RudderStack to Yandex.Metrica.
+---
+
+[Yandex.Metrica](https://metrica.yandex.com/) is a web analytics platform that lets you get an in-depth understanding of your audience's behavior and use the insights to drive business growth.
+
+<div class="infoBlock">
+Find the open source code for this destination in the <a href="">GitHub repository</a>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/yandex-metrica.mdx:10]: @ujjwal-ab can you share the relevant link for this?

Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/yandex-metrica.mdx:10]:
```suggestion
Find the open source code for this destination in the <a href="https://github.com/rudderlabs/rudder-sdk-js/tree/production/integrations/YandexMetrica">GitHub repository</a>.
```

Comment (ujjwal-ab) [reply docs/destinations/streaming-destinations/yandex-metrica.mdx:10]: Yeah, it's correct.

File: docs/destinations/streaming-destinations/yandex-metrica.mdx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+| `properties.name` <br /> `properties.products[index].name` <br/> <span style="color: #4D4DFF;font-size:12px;">Required, if product_id/sku is not present.</span> | `name` |
+| `properties.brand` <br /> `properties.products[index].brand` | `brand` |
+| `properties.category` <br /> `properties.products[index].category` | `category` |
+| `properties.coupon` <br /> `properties.products[index].coupon` | `coupon` |
+| `properties.position` <br /> `properties.products[index].position` | `position` |
+| `properties.price` <br /> `properties.products[index].price` | `price` |
+| `properties.quantity` <br /> `properties.products[index].quantity` | `quantity` |
+| `properties.variant` <br /> `properties.products[index].variant` | `variant` |
+
+The following mappings are applicable only for the `Order Completed` event:
```
Comment (ujjwal-ab) [inline docs/destinations/streaming-destinations/yandex-metrica.mdx:133]:
```suggestion
The following mappings are applicable only for the `purchase` event:
```

File: docs/destinations/streaming-destinations/yandex-metrica.mdx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+});
+```
+
+### Event mapping
+
+The following table lists the default mappings between the RudderStack events and the Yandex.Metrica events:
+
+| RudderStack property | Yandex.Metrica property | Description |
+| :--------------| :---------| :-----|
+| `Product Viewed` | `detail` | View the full description of the product. |
```
Comment (ujjwal-ab) [inline docs/destinations/streaming-destinations/yandex-metrica.mdx:110]:
```suggestion
| `Product Viewed` | `detail` | View the full description of the product. |
| `Product List Viewed` | `detail` | View the full description of the product. |
```

**Review by ujjwal-ab — APPROVED** [review]

File: docs/destinations/streaming-destinations/yandex-metrica.mdx [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+<img src="../../assets/event-stream-destinations/yandex-metrica-container-name.png" alt="Yandex.Metrica container name" />
+
+### Where can I find my Yandex.Metrica tag ID?
+
+To obtain your Yandex.Metrica tag ID, follow these steps:
+
+1. Log into your [Yandex.Metrica](https://metrica.yandex.com/) dashboard. 
+2. From the left sidebar, click **Visitors**. You can see the tag ID listed in the dashboard as well as in the URL, as shown:
+
+<img src="../../assets/event-stream-destinations/yandex-metrica-tag-id.png" alt="Yandex.Metrica tag ID" />
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/yandex-metrica.mdx:184]: Should we blur the ID's in the screenshots?

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/streaming-destinations/yandex-metrica.mdx (hunk 4 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/yandex-metrica.mdx:184]: i think that's fine, it's not sensitive info :)

**Pull request merged**